\chapter{Introduction}
The origin of body plans is an age-old question. Chordates are distinguished body plan is fairly conserved across the phyla \cite{nishida_cell_2014}. Most chordates form tadpole-shaped larvae that commonly have a hollow dorsal neural tube and a post-anal tail containing a central notochord flanked by bilateral muscle. During embryogenesis, the notochord serves as a patterning signal for neural tube and paraxial mesoderm, an addition to the axial skeleton for the larval tail \cite{jeffery_evolution_1999,stemple_structure_2005}. Tunicates are one of the three subphyla of chordates and are so grouped because of their outer covering known as a tunic, and during development form the typical tadpole larval body plan \cite{huber_evolution_2000}. Ascidians and larvaceans are two of the groups within the Tunicates, and both develop typical chordate body plans. The ascidians exhibit their tailed body form during their larvae free-swimming stage, using the tail for locomotion  before becoming sessile, undergoing metamorphosis and filter feeding for the remainder of their lives. During the free-swimming larval stage, the elongation and mobility of the tail is dependent upon the proper formation of the notochord and muscle cells \cite{satoh_ascidian_2003}. In ascidians and in lower vertebrates the improper formation of the notochord leads to severely shortened larva that cannot swim or feed properly \cite{di_gregorio_tail_2002,jiang_ascidian_2005,stemple_structure_2005}. Out of  \mytilde3000 ascidian species, 16 are known to have independently lost their larval tail, differentiated notochord, muscle cell, and other chordate features, changing their developmental body plan. The majority of these losses have happened within the \textit{Molgula} clade \cite{berrill_studies_1931,swalla_interspecific_1990}, although the Styelidae also have two identified tail-less species \cite{huber_evolution_2000}.

Ascidians are a simple system in which to study developmental processes: their cell lineages have been traced starting at fertilization \cite{nishida_cell_1983} to gastrulation \cite{nishida_cell_1985,nishida_cell_1987}, they have invariant early cell lineages and a small number of cells \cite{lemaire_evolutionary_2011}, and there has been no documentation of ascidians developing without an invariant cell lineage.  Ascidian development is nearly identical across distantly related solitary ascidians \cite{lemaire_ascidians_2008,nishida_cell_2014}. They also have rapid embryogenesis, compact genomes, and simplified larval body plans containing few larval tissue types \cite{corbo_characterization_1997,jeffery_minireview_2002,dehal_draft_2002}.  %In \textit{Ciona intestinalis} there are ~2,600 cells, with 36 muscle cells and 40 notochord and the time line \cite{}. 
In addition to the several Molgulids that have independently lost their tail, two Molgulids, one tailed and one tail-less species, can be hybridized, offering the opportunity to study the mechanism behind evolutionarily divergent body plans \cite{jeffery_evolutionary_1991}. %Although \textit{M. occulta} and \textit{M. oculata} present great systems evolutionarily to study tail development and loss, they have several shortcomings as experimental models: they are only found on the Northern coast of France and have yet to be cultured, they only spawn for one month out of the year, and many of the molecular techniques used in other ascidians have not yet been optimized for these two species. Thus tunicates are good models for studying notochord specification.

Many genes in the notochord gene network have been identified by subtractive hybridization screening and microarrays \cite{jeffery_factors_1992,hotta_characterization_2000,gyoja_analysis_2007,kobayashi_differential_2013}. More recently, sequencing technologies such as Ion Torrent, Roche 454 and Illumina have made genome or transcriptome wide analysis more readily available for non-model species. These technologies have several advantages over microarrays: they have a wider scope, are more precise and are able to find novel genes \cite{marioni_rna-seq:_2008}. With the advances in technology we have now sequenced the transcriptomes of both species and their hybrid. This ability to sequence the entire transcriptome at a high resolution, allows us to look at pivotal time points in tail development and compare across closely related species studying mechanisms that have been lost or modified during evolution. 

We present a comparative study of the tailed \textit{M. oculata} and the tail-less \textit{M. occulta} through gene expression in order to understand the underlying factors behind tail development, tail loss and the origin of the chordate body plan. Although this study presents the first assembled \textit{Molgula} genomes, there are a number of sequenced tunicate genomes available: in particular, we use the assembled and annotated genome of \textit{Ciona intestinalis}, which serves as the most documented and closest complete reference for the \textit{Molgula} and other ascidian species \cite{dehal_draft_2002,satoh_ascidian_2003,satoh_ciona_2003}. We began this project with RNA-seq data from several time points from each of the species (\textit{M. occulta} and \textit{M. oculata}) and their hybrid. 
%Next-generation sequencing (NGS) is an effective method of producing observations and generating hypotheses to be tested experimentally.
Whole genome and transcriptome sequencing will not definitively identify the factors involved in the development of the tailed chordate body plan since other processes are occurring, but examining the correlation of gene expression patterns between the tailed, tail-less and interspecific \textit{Molgula} hybrid allows us to filter genes associated with this process. Before we can make biological inferences from our data we have to produce a quality assembly, because of this we first assessed the quality of an efficient low-memory assembly pipeline for our RNA-seq data.
%and identified quality metrics other than N50 and contig length, since these are not best metrics for assessing transcriptome quality \cite{oneil_assessing_2013}.
We later obtained genomic DNA and assembled the genomes of \textit{M. occulta}, \textit{M. oculata} and a more divergent species \textit{M. occidentalis}. This allowed us to analyze the homology between ascidian gene networks, and build more complete transcript modules for differential expression \cite{vijay_challenges_2012}.

