\chapter{Conclusions and Discussion}

The chordate body plan is conserved throughout the phyla with only a few species deviating from the conserved body plan; at some point in their life, most chordates develop a tadpole larvae containing a hollow dorsal neural tube, and a postanal tail containing a notochord flanked by muscle cells. However, it has been documented that 16 species out of the approximately 3000 species of tunicates\textemdash one of the three subphyla of chordates\textemdash have independently undergone tail loss, with the majority being within the molgulids \cite{berrill_studies_1931,huber_evolution_2000}. Because they have both tailed and tail-less species, molgulids are useful models for studying changes in the development of body plans. Having closely related species with altered development allows us to look at the short-term modifications that occurr during the evolution of alternate body plans, showing us that larval development can evolve rapidly. Of particular interest, two molgulids have the ability to hybridize, which allows us to examine the mechanisms of evolutionary change at an allele-specific level.

\section{Evaluating a lightweight transcriptome assembly pipeline on two closely related ascidian species}
The drop in sequencing price has aided in our efforts to understand body plan development by allowing us to quickly and inexpensively obtain transcriptome and genome sequences, along with temporal expression profiles. However, for non-model systems with no existing genomic sequence, we must first assemble the transcriptomes and genomes.  The methodology for assembly is not unambiguous: there are a number of different steps\textemdash quality trimming, filtering, and choice of assembler(s), with several programs at each step and no clear choice. Studies have been done to compare assemblers, but no one assembler is clearly the best choice \cite{clarke_comparative_2013}. Many times assembly methods are chosen on the usability of software and the availability of resources.

Factors that are most limiting for assemblies are memory, and in some cases time. The redundant nature of sequencing data allows for the removal of redundant reads with approaches like digital normalization, but at what cost? Here in efforts to assemble the first two \textit{molgula} transcriptomes, we evaluate the cost of filtering sequencing reads for redundancy. We show that abundance filtering quality trimmed assembly reads enables transcriptome assembly with a reduction in both memory requirement and assembly time, while retaining essentially the same information content (e.g. number of genes, average gene length, and homology to the closest sequence species). We have demonstrated that the Oases and Trinity assemblers return similar results, both suitable for downstream analysis using the full or redundancy filtered dataset. Our pipelines are available so we also provide methodology to be used by future researchers. In addition assembly metrics to evaluate assemblies are also an important step in the protocol. One of the standard metrics for evaluating assemblies is the N50, but this is designed for genomic evaluation and does not clearly translate to transcriptomes because of isoforms. For example, the same exon may be included in multiple splice variants, inflating the total bp assembled; some assemblers may choose to report more isoforms than others even with the same read support; and N50 makes little sense for transcriptomes. In contrast, homology and programs such as CEGMA are more informative and useful for downstream analysis, because they measure the information in the transcriptome and cover the annotation step as well. When measuring recovered homologies and CEGMA measurements we find that both assemblers and both treatments compare well. %When only taking N50 into account 

\section{Change in gene function and cis-regulatory binding sites}
In tunicates \textit{hox} genes are not important for patterning along the anterior-posterior axis, as they are in vertebrate and other bilaterians \cite{finnerty_origins_2003,mallo_regulation_2013,ikuta_limited_2010}. This limited function in the \textit{hox} genes is evident from the lack of noticeable phenotypic defects in the knockdown of \textit{C. intestinalis hox} genes, with the exception of \textit{hox10} and \textit{hox12}. \textit{Hox10} is involved in neuronal development and \textit{hox12} is involved in the formation of the posterior most portion of the the tunicate larval tail \cite{ikuta_limited_2010}. In addition, tunicates have undergone loss and rearrangement within the \textit{hox} clusters, unlike most animals that have been studied\cite{ikuta_organization_2005}. There have yet to be two tunicate species found with the same \textit{hox} cluster configuration - there are typically changes in ordering, duplications, and which hox genes are present (figure 4.2). Only \textit{C. intestinalis} has enough scaffolding to completely identify the structure and intergenic spacing between the \textit{hox} genes. However, of the \textit{hox} gene we found on the same scaffolds in \textit{M. occulta}, \textit{M. oculata} and \textit{M. occidentalis} the average intergenic spacing is far smaller than in {\em Ciona}, ranging from 10-25 kb in length. Without additionally scaffolding we cannot fully examine the rearrangements within the \textit{hox} clusters of ascidians based on the molgulid sequences. Perhaps soon after diverging from vertebrae and cephalochordates, the \textit{hox} cluster genes were co-opted into other GRNs, leading to higher lability than in species where they retained their stereotyped roles.

We have also identified a number of enhancers between \textit{C. intestinalis}, \textit{M. occidentalis}, and \textit{M. oculata} that were either mutually unintelligible, or asymmetrically intelligible. We propose that a high frequency of compensatory changes are required for the rapidly evolving ascidians to maintain their invariant embryonic cell lineages in the presence of their compact genomes. These compensatory changes have caused the enhancers to diverge within the ascidians while allowing them to develop with their typical body plan.

\section{Differential expression analysis of tail loss in an invertebrate chordate}
Studying closely related organisms gives us insight into the underlying evolutionary mechanisms of divergence and underlying development. Indirect and direct developing sea urchins have been studied showing that change in axial cleavage patterns and early cell fate leads to divergent larval body plans that can exhibit similar adult body plans \cite{wray_evolutionary_1989,henry_evolutionary_1990}. In contrast, ascidians retain their typical cell division and invariant cell fate in direct, indirect, tailed and tail-less ascidians \cite{jeffery_evolutionary_1991,maliska_molgula_2010}. 
   
We have shown that evolutionary drift has occurred in the cis-regulatory modules of developing ascidian embryos between \textit{C. intestinalis} and \textit{M. occidentalis}, and these enhancers are not always capable of driving expression in other ascidian species. It appears that the same has happened between the tailed \textit{M. oculata} and tail-less \textit{M. occulta}, however those cis-mechanisms are restored in the (\textit{M. oculata x M. occulta}) interspecific hybrids, probably allowing transcription factors to bind their targets at a higher affinity and restoring the necessary level of expression to develop the urodele features. Further research is needed to identify and test cis-elements, but as of now we have a better understanding behind the mechanism of the restoration of the larval tail in hybrids, which we can at least hypothesize is driven by recovery of cis-regulatory modules.

\section{Conclusion}

The identification of regulatory genes expressed differently in the urodele and anural species is needed to understand the molecular mechanism underlying the evolutionary transition from urodele to anural development. When examining the tailed and tail-less \textit{Molgula}\textemdash \textit{M. oculata} and \textit{M. occulta}\textemdash there is a strong overlap (91\%) in the expression of genes that can be annotated using the \textit{C. intestinalis} proteom. There was also a strong overlap in genes associated with notochord development, which is the key structure-defining element not present in \textit{M. occulta}. We observed, it was not the lack of genes in the transcriptome, but the expression of said genes at a sufficient level, which has led to the tail-less phenotype. One of the shortcomings of our study is the lack of replicates from all stages; ideally we would like at least three per developmental stage for each organism and because of this there are likely to be more false positives and false negatives in our analysis than there would be with more replicates. However, we were able to identify differentially expressed genes using the available replicates to estimate the dispersion for each of the samples, and able to identify a subset of genes involved the development of the urodele features by examining the correlation between two independent datasets, the tailed \textit{M. oculata} and the hybrid. For further examination replicates are needed, along with addition sequencing of other tailed and tail-less molgula genomes and transcriptomes. This would give us stronger insights into why the molgula are able to easily undergo tail-loss and the toolkit for molgula tail development. 

Here we were able to see in the hybrids that the restored expression came from the tailed allele and we proposed that cis-regulatory modules are the cause of this. The restored expression of genes involved with the development of urodele features are acting an allele specific manner, while expression for other genes (total) has a more balanced expression from each allele. The tailed allele restored the enhancer binding sites; presumably TF were able to bind at a higher affinity and restore necessary level of expression to generate urodele features, including the formation of the notochord and hence the larval tail. To confirm this hypothesis we would first perform a ChIP-seq analysis to identify direct binding targets, and correlate them with the differentially expressed genes. A clear starting point is the gene \textit{bra} which is known to be important for notochord development, and \textit{Tbx2/3} which is downstream of \textit{bra}, and involved in convergence and extension \cite{katikala_functional_2013}.  {\em Tbx2/3} was also identified in our differential expression analysis. From this gene-targeted analysis we can examine the divergence between \textit{M. occulta} and \textit{M. oculata} enhancers, examine their binding affinities, and test the found sites using transgenesis to see if we can restore the \textit{M. occulta}'s tail.

Our hypothesis depends heavily on the assumption that the gene regulatory network (GRN) is conserved within the tunicates to understand the mechanisms behind tail loss, and the GRN is conserved throughout the chordates to understand the origin the chordate body plan. Comparing the GRN of distantly related organisms can identify the genes necessary to the development of similar phenotypes and body plans. Currently we plan to expand our analysis by comparing the \textit{molgula} to the ten tunicate genomes assemblies found on the aniseed website (\url{http://www.aniseed.cnrs.fr/}), and available chordate genomes found on NCBI (\url{http://www.ncbi.nlm.nih.gov/}) and ensembl (\url{ensembl.org}). With the information we currently have we will give deeper look into the sequence divergence of both coding and non coding regions. To examine the noncoding regions addition genomic sequencing is needed for gap filling and scaffolding. However, it is necessary to determine if the GRN have conserve structure by e.g. confirming the spatial expression patterns for the network. Next steps on this project would identify relevant transcription factors by annotating the genes and then selecting differentially expressed genes associated with DNA binding functions. Once we identify our candidate TFs we will proceed with whole mount in situ hybridization to confirm conserved spatial patterning.

