\chapter{Conclusions and Discussion}
The chordate body plan is conserved throughout the phyla with several species deviating; most chordates develop a tadpole larvae containing a hollow dorsal neural tube, and a postanal tail containing a notochord flanked by muscle cells at some point in their life cycle. %Our work gives insight into the mechanisms behind the chordate body plan. 
Molgulids are especially useful models for studying changes in the development of body plans because they have both tailed and tail-less species, two of which have the ability to produce interspecific hybrids \cite{swalla_interspecific_1990}. This shows us that larval development can evolve rapidly, and allows us to examine the mechanisms of evolutionary change at an allele-specific level.

\section{Evaluating a lightweight transcriptome assembly pipeline on two closely related ascidian species}
The drop in sequencing price has aided in our efforts to understand body plan development. %Many non-model systems are now being sequenced with the drop in sequencing price. 
The methodology for assembly is not cut and drier: there are a number of different steps\textemdash quality trimming, filtering, and choice of assembler(s), with several programs at each step and no clear choice. Studies have been done to test which assembler is best, in which it was determined no one assembler is the best \cite{clarke_comparative_2013}. Many times assembly methods are chosen on the usability of software and the availability of resources. Factors that are most limiting for assemblies are memory, and in some cases time. The redundant nature of sequencing data allows for the removal of redundant reads, but at what cost? Here in efforts to assemble the first two molgula transcriptomes, we evaluate the cost of redundancy filtered sequencing reads. We show that pre-filtering quality trimmed assembly reads does not reduce the information content of the assembly, but a reduction in computational resources. We have demonstrated that the Oases and Trinity assemblers return similar results, both suitable for downstream analysis using the full or redundancy filtered dataset. Our pipelines are available so we also provide methodology to be used by future researchers. In addition assembly metrics to evaluate assemblies are also an important step in the protocol. One of the standard metrics for evaluating assemblies is the N50, which has been designed for genomes and does not clearly translate to transcriptomes because of their fragmented nature, and the likelihood of chimeric contigs; homology and programs such as CEGMA are more informative and useful for downstream analysis. %When only taking N50 into account 
 
The two \textit{Molgula} species that were assembled are evolutionarily interesting because of the position within the chordates and \textit{M. occulta}'s divergent body plan. The downstream analysis of these two molgula tanscriptomes will give insight into the formation of the chordate body plan. 

\section{Genome assembly and characterization}
In tunicates \textit{hox} genes are not important for patterning along the anterior-posterior axis, as they are in vertebrate and other bilaterians \cite{finnerty_origins_2003,mallo_regulation_2013,ikuta_limited_2010}. This limited function of the \textit{hox} genes is evident, from the loss and rearrangement of several genes in the ascidian \textit{hox} clusters, in addition to no noticeable phenotypic defects in the knockdown of \textit{C. intestinalis hox} genes, with the exception of \textit{hox10} and \textit{hox12}. \textit{Hox10} is involved in neuronal development and \textit{hox12} is involved the formation for the posterior most portion of the the tunicate larval tail \cite{ikuta_limited_2010}. There have yet to be two tunicate species studied with the same \textit{hox} cluster configuration. Perhaps after diverging from vertebrae and cephalochordates, the inconsistency in the function of the hox clusters was established in tunicates to conserve their highly invariant  body plan and the function of the \textit{hox} genes  were co-opted by other genes. % leading the hox gene clusters to eventually 

In addition we have identified a number of enhancer between \textit{C. intestinalis}, \textit{M. occidentalis}, and \textit{M. oculata} that were either mutually unintelligible, or asymmetrically intelligible. We propose that these changes are required for the rapidly evolving ascidians to accommodate the constraints imposed by their invariant embryonic cell lineages and highly compact genomes.

\section{Differential expression analysis of tail loss in an invertebrate chordate}
Studying closely related organisms gives us insight into the underlying evolutionary mechanisms of divergence and underlying development. Indirect and direct developing sea urchins have been studied showing that change in axial cleavage patterns and early cell fate leads to divergent larval body plans that can exhibits similar adult body plans \cite{wray_evolutionary_1989,henry_evolutionary_1990}. In contrast, ascidians retain their typical cell division and invariant cell fate in both direct, indirect, tailed and tail-less ascidians \cite{jeffery_evolutionary_1991,maliska_molgula_2010}. %Next generation sequencing is a great way to study non-model species. A broad swath of information can be gained from both RNA and DNA sequencing. 
   
We have shown that evolutionary drift has occurred in the cis-regulatory and are not always capable of driving expression in other ascidian species. It appears that the same has happened between the tail-less \textit{M. occulta}, however those cis mechanisms are restored in the (\textit{M. oculata x M. occulta}) interspecific hybrids. Further research is needed to identify and test cis elements, but as of now we have a better understanding of the mechanism behind different larval body plans in the Molgulids. 

\section{Conclusion}

Identification of regulatory gene expressed differently in the urodele and anural species is needed to understand the molecular mechanism underlying the evolutionary transition from urodele to anural development. We identifed genes that may be involved in this developmental body plan transition from tailed to tail-less, however, without replicates we cannot call differential expressed genes with statistical confidence, and there are sure to be both false positives and false negatives in our analysis. However, examining the correlation between the tailed \textit{M. oculata} and the hybrid, we were able to identify a subset of genes that we believe to be true differentially expressed genes. Replicates are needed, and addition sequencing of other tailed and tail-less \textit{molgula} genomes and transcripotmes would give us a better insight into why the \textit{molgula} are able to easily undergo tail loss and the toolkit for \textit{molgula} tail development. 

Our hypothesis depends heavily on the assumption that the gene regulatory network (GRN) is conserved within the tunicates to understand the mechanisms behind tail loss, and the GRN is conserved throughout the chordates to understand the origin the chordate body plan. Currently we plan to expand our analysis by compare the \textit{molgula} to the ten tunicate genomes assemblies found on the aniseed website (\url{http://www.aniseed.cnrs.fr/}), and available chordate genomes found on NCBI (\url{http://www.ncbi.nlm.nih.gov/}) and ensembl (\url{ensembl.org}). Furthermore, molecularly, we will examine the spatial expression to confirm orthologous function.  Additionally we would like to identify binding sites using Chip-seq and examine the found sites using transgenesis to see if we can restore the \textit{M. occulta's} tail.


---- Thinking


 The fact that the zygotic genome restore sty urodel features and the expression come from the zygotic allele leads us to believe that cis-acting factors are  a major cause of tail loss. we need to closer examine the noncoding regions associated with our identified set of gees, in addition to examining the divergence of these sequences. We have increased the information known about the mechanisms behind tail loss, identifying additional genes and better understanding the mechanisms behind the regulation of these genes. One of the major shortcomings of our analysis is a lack of replicates for the RNA-seq samples.   In addition to replicates, sequencing of the fertilized and unfertilized embryo will help decipher maternal and zygotic effects.
 For these reasons further comparisons are needed. With the information we currently have a deeper look into the sequence divergence of both gene and non coding regions. To examine the noncoding regions addition genomic sequencing is needed for gap feeling and scaffolding.
because genes can have multiple roles or do not always have the orthologous functions so spatial expression is needed.
Although there is need for additional computational analysis, we can now began to 
chip-seq to determine the interactions
knock down



identify less board regulation regions, and perhaps identify the causes for lose in across species regulation.

The sequencing of these organisms adds to the body of knowledge allowing ups to examine a wider range of chordates and the similarities and drast differences. 

Differential expression is a known cause for different phenotypes, but the mechanisms are not always understood. In this case change in the cis-regulatory elements appears to have lead to differential expression.

 we can start by examining the available data, genomes on Aniseed, NCBI. ultimately more mogul and tunicate genomes would be needed to identity what is unique about the molgula that allows them to loss their tail so easily.  
